source/extensions/filters/http/ext_proc/processor_state.cc

```diff
@@ -295,7 +295,7 @@ absl::Status ProcessorState::handleBodyResponse(const BodyResponse& response) {
           const auto mut_status = MutationUtils::applyHeaderMutations(
               common_response.header_mutation(), *headers_,
               common_response.status() == CommonResponse::CONTINUE_AND_REPLACE,
-              filter_.config().mutationChecker(), filter_.stats().rejected_header_mutations_);
+              filter_.config().mutationChecker(), filter_.stats().rejected_header_mutations_, body_mode_ == ProcessingMode::STREAMED);
```

Comment:
Same as above, here the last parameter can be " true".

And below line 309 can be removed.